lib/sqlalchemy/pool/impl.py

```diff
@@ -182,6 +182,9 @@ def status(self):
     def size(self):
         return self._pool.maxsize
 
+    def timeout(self):
+        return self._pool._timeout
```

Comment:
**mike bayer** (zzzeek) wrote:

_timeout is on "self", so need to make this "self._timeout".